sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_events_v1/query.sql

```diff
@@ -0,0 +1,572 @@
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN notification_status_authorized > 0
+          THEN client_id
+      END
+    ) AS notification_status_authorized_users,
+    SUM(notification_status_notDetermined) AS notification_status_notDetermined,
+    SUM(notification_status_denied) AS notification_status_denied,
+    SUM(notification_alert_setting_notSupported) AS notification_alert_setting_notSupported,
+    SUM(notification_alert_setting_disabled) AS notification_alert_setting_disabled,
+    SUM(notification_alert_setting_enabled) AS notification_alert_setting_enabled
+  FROM
+    product_features
+  WHERE
+    submission_date >= start_date
+  GROUP BY
+    1
+)
+SELECT
+  d.submission_date,
+  dau,
+    /*Credit Card*/
+  cc_autofill_failed,
+  cc_autofill_failed_users,
+  cc_autofill_settings_tapped,
+  cc_autofill_settings_tapped_users,
+  cc_autofill_toggle,
+  cc_autofill_toggle_users,
+  cc_autofilled,
+  cc_autofilled_users,
+  cc_form_detected,
+  cc_form_detected_users,
+  cc_save_prompt_create,
+  cc_save_prompt_create_users,
+  cc_sync_toggle,
+  cc_sync_toggle_users,
+    /*History*/
+  history_delete_tap,
+  history_delete_tap_users,
+  history_opened,
+  history_opened_users,
+  history_removed,
+  history_removed_users,
+  history_removed_all,
+  history_removed_all_users,
+  history_removed_today,
+  history_removed_today_users,
+  history_removed_today_and_yesterday,
+  history_removed_today_and_yesterday_users,
+  history_search_tap,
+  history_search_tap_users,
+    /*FxA*/
+  fxa_disconnect,
+  fxa_disconnect_users,
+  fxa_login_completed_view,
+  fxa_login_completed_view_users,
+  fxa_login_token_view,
+  fxa_login_token_view_users,
+  fxa_login_view,
+  fxa_login_view_users,
+  fxa_paired,
+  fxa_paired_users,
+  fxa_registration_code_view,
+  fxa_registration_code_view_users,
+  fxa_registration_completed_view,
+  fxa_registration_completed_view_users,
+  fxa_registration_view,
+  fxa_registration_view_users,
+  fxa_use_email,
+  fxa_use_email_users,
+    /*Privacy*/
+  private_browsing_button_tapped,
+  private_browsing_button_tapped_users,
+  private_browsing_button_tapped_enter_private_mode,
+  private_browsing_button_tapped_enter_private_mode_users,
+  private_browsing_icon_tapped,
+  private_browsing_icon_tapped_users,
+  app_icon_new_private_tab_tapped,
+  app_icon_new_private_tab_tapped_users,
+  tabs_tray_new_private_tab_tapped,
+  tabs_tray_new_private_tab_tapped_users,
+    /*Awesomebar Location*/
+  drag_location_bar,
+  drag_location_bar_users,
+  location_top,
+  location_top_users,
+  location_bottom,
+  location_bottom_users,
+    /*Notification*/
+  notification_status_authorized,
+  notification_status_authorized_users,
+  notification_status_notDetermined,
+  notification_status_denied,
+  notification_alert_setting_notSupported,
+  notification_alert_setting_disabled,
+  notification_alert_setting_enabled
+FROM
+  dau_segments d
+LEFT JOIN
+  product_features_agg p
+ON
```

Comment:
For consistency with the other query files, we should consider using `USING()` here for joining the two datasets.